posttraining_evals.sh

```diff
@@ -0,0 +1,576 @@
+# Runs four evaluations: IFEval, MTBench, AlpacaEval and MultiIF
+# Usage: ./post_training_evals.sh MODEL_PATH [LANGUAGE] [EVAL]
+
+set -e  # Exit on any error
+
+# Check if model argument is provided
+if [[ $# -eq 0 ]]; then
+    echo "Usage: $0 MODEL_PATH [LANGUAGE] [EVALUATION...]"
+    echo ""
+    echo "Arguments:"
+    echo "  MODEL_PATH     Path to the model directory (required)"
+    echo "  LANGUAGE       Language code for evaluations (optional, default: en)"
+    echo "  EVALUATION     Specific evaluation(s) to run (optional, default: all)"
+    echo ""
+    echo "Available evaluations:"
+    echo "  ifeval       - Instruction Following Evaluation"
+    echo "  mtbench      - Multi-Turn Benchmark"
+    echo "  alpaca_eval  - AlpacaEval Evaluation"
+    echo "  multi_if     - Multi-IF Evaluation"
+    echo ""
+    echo "Examples:"
+    echo "  $0 /path/to/my/model                    # Run all evaluations"
+    echo "  $0 /path/to/my/model fi                 # Run all evaluations in Finnish"
+    echo "  $0 /path/to/my/model en ifeval mtbench  # Run only IFEval and MTBench"
+    echo "  $0 /path/to/my/model en alpaca_eval     # Run only AlpacaEval"
+    exit 1
+fi
+
+# Configuration
+PROJECT="project_462000353"
+MODEL="$1"  # Model path from command line argument
+STATE_LOG="evals_state.json"
+LOGS_DIR="logs"
+WORKSPACE_ROOT=$(pwd)/posttraining
+
+# Parse arguments more flexibly
+LANGUAGE="en"  # Default language
+SELECTED_EVALS=()
+
+export HF_HOME="${HF_HOME:-/scratch/project_462000353/hf_cache}"
+
+# Parse remaining arguments
+for arg in "${@:2}"; do
+    case "$arg" in
+        ifeval|mtbench|alpaca_eval|multi_if)
+            SELECTED_EVALS+=("$arg")
+            ;;
+        en|fi|sv|da|no|is)  # Common language codes
+            LANGUAGE="$arg"
+            ;;
+        *)
+            # Check if it looks like a language code (2-3 letters)
+            if [[ ${#arg} -le 3 && "$arg" =~ ^[a-z]+$ ]]; then
+                LANGUAGE="$arg"
+            else
+                echo "Error: Unknown argument '$arg'"
+                echo "Expected: language code (e.g., en, fi) or evaluation name (ifeval, mtbench, alpaca_eval, multi_if)"
+                exit 1
+            fi
+            ;;
+    esac
+done
+
+# If no specific evaluations provided, run all
+if [[ ${#SELECTED_EVALS[@]} -eq 0 ]]; then
+    SELECTED_EVALS=("ifeval" "mtbench" "alpaca_eval" "multi_if")
+fi
+
+# Check for required environment variables for MTBench and AlpacaEval judge phases
+if [[ -z "$OPENAI_API_KEY" ]]; then
+    echo "Warning: OPENAI_API_KEY environment variable not set"
+    echo "This is required for MTBench and AlpacaEval judge phases"
+    echo "Set it with: export OPENAI_API_KEY=your_key_here"
+fi
+
+if [[ -z "$OPENAI_ORG_ID" ]]; then
+    echo "Warning: OPENAI_ORG_ID environment variable not set"
+    echo "This is required for MTBench and AlpacaEval judge phases"
+    echo "Set it with: export OPENAI_ORG_ID=your_org_id_here"
+fi
+
+# Repository URLs
```

Comment:
would be good to reference a specific commit/tag in each url so we run the same code every time